tests/serializers/test_serializers.py

```diff
@@ -56,3 +56,8 @@ def test_no_children(self):
         data = self.get_json_response(url)
 
         self.assertEqual(len(data['first']), 0)
+
+    def test_nested_urls(self):
+        url = reverse('parent3-list')
+        data = self.get_json_response(url)
+        print(data)
```

Comment:
True. It does however fail when you try to serialize the URLs without my Nested Serializer. I will work on improving this test though.